www/web-proxy-useracl/Makefile

```diff
@@ -0,0 +1,9 @@
+PLUGIN_NAME=		web-proxy-useracl
+PLUGIN_VERSION=		0.0.1
+PLUGIN_COMMENT=		Group & User Squid ACL
+PLUGIN_DEPENDS=		os-web-proxy-sso
```

Comment:
why is that? if you depend on your own plugin, shouldn't this code be in the other plugin as well? :)